### DISCUSSION TITLE
Windows Active Directory authentication

### DISCUSSION THREAD

**Discussion by danielpsc** [Budibase/budibase#4669] [Help]
Hi, just set up Budibase to try it out. I see OIDC is supported as an auth method and can be tied to AzureAD. What about plain old on-prem Active Directory, is that supported as an auth method for enduser login, so they can just enter their existing Windows username/password?

**Comment by mjashanks** [comment]
Hey @danielpsc - I'm not that familiar with on-prem AD. The answer is - if it supports OIDC then yes, otherwise no!

**Reply by danielpsc** [reply]
Ok thank you. From some googling it appears that OIDC is supported on Windows Server 2016 and on, which we are not (yet) running. So I guess this will not work for us. Thank you though!
